Jenkinsfile

```diff
@@ -34,23 +34,30 @@ pipeline {
             parallel {
                 stage('Build Docs') {
                     agent { label "docker" }
-                    environment { XMOSDOC_VERSION = "v4.0" }
+                    environment { XMOSDOC_VERSION = "v5.0" }
```

Comment:
Not worth the Jenkins rerun, but would suggest either 5.1.2 or just "latest" here - we'll pick this up in a new Jira story though!

Reply:
Indeed! 
As a side suggestion for another PR: there is a Jenkins machine with latex, so I recommend changing this docker command to use directly pip works nicely:
https://github.com/xmos/lib_camera/blob/4f2eab1eb348f8dbdd308c2f3fe96fce7a919c95/Jenkinsfile#L15
https://github.com/xmos/lib_camera/blob/4f2eab1eb348f8dbdd308c2f3fe96fce7a919c95/Jenkinsfile#L122